### ISSUE TITLE
Visible navigation elements are hidden for screen-readers

### ISSUE THREAD

**Issue by JuliaGru** [FrDH/mmenu-js#1226]
I came across an accessiblity bug in the browsers Edge & Chrome, due to wrong parent-elements.
As soon as these are "aria-hidden="true" the navigation elements are not visible for screen reader users anymore.
Here an example:
![grafik](https://user-images.githubusercontent.com/25286061/116966964-9beb2580-acb1-11eb-8128-c4062cf9893b.png)

**Comment by FrDH** [comment]
Thank you for reporting the issue.
I had a look and it is actually the navbar in one of the submenus that points to the main menu (with aria-owns).
I'll see if I can fix this shortly.

**Comment by FrDH** [comment]
Try the new release 8.5.23
Unfortunately I had to remove the aria-owns attribute for now.
